pkg/ui/workspaces/cluster-ui/src/jobs/jobsPage/jobsTable.tsx

```diff
@@ -227,43 +225,6 @@ export function makeJobsColumns(): ColumnDescriptor<Job>[] {
       sort: job => TimestampToMoment(job?.finished).valueOf(),
       showByDefault: true,
     },
-    {
-      name: "lastExecutionTime",
-      title: (
-        <Tooltip
-          placement="bottom"
-          style="tableTitle"
-          content={<p>Date and time of the last attempted job execution.</p>}
-        >
-          <>
-            {jobsColumnLabels.lastExecutionTime} <Timezone />
-          </>
-        </Tooltip>
-      ),
-      cell: job => (
-        <Timestamp
-          time={TimestampToMoment(job?.last_run, null)}
-          format={DATE_WITH_SECONDS_AND_MILLISECONDS_FORMAT}
-        />
-      ),
-      sort: job => TimestampToMoment(job?.last_run).valueOf(),
-      showByDefault: true,
```

Comment:
I suppose an alternative is to flip this to false so it is still available for custom column selection.

Reply:
I started with that, but as mentioned in the commit message, I think drawing any attention to these columns is bad, since for almost all jobs they're false and actively misleading: the job could be on its 100th retry inside its resumer but this will happily be reporting 1.

Reply:
I think we need to be removing this facility from the jobs _system_ altogether but in the meantime the less anyone sees it and is confused by it the better.